pkg/hubble/relay/observer/server.go

```diff
@@ -28,6 +30,8 @@ import (
 // Reporting up to 10 unavailable nodes is probably reasonable.
 const numUnavailableNodesReportMax = 10
 
+const heartbeatInterval = time.Minute
```

Comment:
nit: This seems to be quite a lightweight code path to execute (check and report status). I believe it makes sense to call this method more often, i.e. every 5 seconds.